i18n/en.json

```diff
@@ -4942,22 +4938,22 @@
-  "user.settings.security.passwordErrorUppercaseNumber": "Your password must contain between {min} and {max} characters made up of at least one uppercase letter and at least one number.",
-  "user.settings.security.passwordErrorUppercaseNumberSymbol": "Your password must contain between {min} and {max} characters made up of at least one uppercase letter, at least one number, and at least one symbol (e.g. \"~!@#$%^&*()\").",
-  "user.settings.security.passwordErrorUppercaseSymbol": "Your password must contain between {min} and {max} characters made up of at least one uppercase letter and at least one symbol (e.g. \"~!@#$%^&*()\").",
+  "user.settings.security.passwordError": "Must be {min}-{max} characters long.",
+  "user.settings.security.passwordErrorLowercase": "Must be {min}-{max} characters long and include lowercase letters.",
+  "user.settings.security.passwordErrorLowercaseNumber": "Must be {min}-{max} characters long and include lowercase letters and numbers.",
+  "user.settings.security.passwordErrorLowercaseNumberSymbol": "Must be {min}-{max} characters long and include lowercase letters, numbers, and special characters.",
+  "user.settings.security.passwordErrorLowercaseSymbol": "Must be {min}-{max} characters long and include lowercase letters and special characters.",
+  "user.settings.security.passwordErrorLowercaseUppercase": "Must be {min}-{max} characters long and include both lowercase and uppercase letters.",
+  "user.settings.security.passwordErrorLowercaseUppercaseNumber": "Must be {min}-{max} characters long and include both lowercase and uppercase letters, and numbers.",
+  "user.settings.security.passwordErrorLowercaseUppercaseNumberSymbol": "Must be {min}-{max} characters long and include both lowercase and uppercase letters, numbers, and special characters.",
+  "user.settings.security.passwordErrorLowercaseUppercaseSymbol": "Must be {min}-{max} characters long and include both lowercase and uppercase letters, and special characters.",
+  "user.settings.security.passwordErrorNumber": "Must be {min}-{max} characters long and include numbers.",
+  "user.settings.security.passwordErrorNumberSymbol": "Must be {min}-{max} characters long and include numbers and special characters.",
+  "user.settings.security.passwordErrorSymbol": "Must be {min}-{max} characters long and include special characters.",
+  "user.settings.security.passwordErrorUppercase": "Must be {min}-{max} characters long and include uppercase letters.",
+  "user.settings.security.passwordErrorUppercaseNumber": "Must be {min}-{max} characters long and include uppercase letters, and numbers.",
+  "user.settings.security.passwordErrorUppercaseNumberSymbol": "Must be {min}-{max} characters long and include uppercase letters, numbers, and special characters.",
+  "user.settings.security.passwordErrorUppercaseSymbol": "Must be {min}-{max} characters long and include uppercase letters, and special characters.",
```

Comment:
@cwarnermm @matthewbirtch  Could you please check that the variations in the password messages above are ok?